### ISSUE TITLE
pyephem  3.7.6.0    make all-pdf   

### ISSUE THREAD

**Issue by oystercatcher** [brandon-rhodes/pyephem#111]
I have installed  pyephem 3.7.6.0 on fedora 24.

I have installed  python3-sphinx, texlive to create documentation for pyephem.

After cd to doc directory,  I did make html which was ok
when I try to make pdf  from  doc/_build/latex
make all-pdf     it errors out.  
```
(/usr/share/texlive/texmf-dist/tex/latex/psnfss/ts1ptm.fd) [5] [6] [7] [8]
[9] [10] [11] [12] [13] [14] [15] [16] [17] [18] [19] [20]

! Package inputenc Error: Unicode char \u8:′ not set up for use with LaTeX.

See the inputenc package documentation for explanation.
Type  H <return>  for immediate help.
 ...                                              

l.1645 the more traditional 21°46′
                                     15.66′′ is not possible
? 
```
I type quit and make ends
```
? quit
OK, entering \batchmodeMakefile:66: recipe for target 'PyEphem.pdf' failed
make: **\* [PyEphem.pdf] Error 1
```
just a note on fedora in addition to texlive, I needed these prereqs.

```
            texlive-titlesec, texlive-framed,
            texlive-threeparttable, texlive-wrapfig,
            texlive-upquote, texlive-capt-of,
            texlive-multirow, texlive-eqparbox
```

**Comment by brandon-rhodes** [comment]
I see one issue where an error like this was blamed on an old version of Sphinx:

https://github.com/Graylog2/documentation/issues/32

but upgrading Sphinx did not fix the problem for me. It looks like Sphinx now supports another way of building PDFs, though:

http://www.sphinx-doc.org/en/1.4.8/builders.html

> Note that a direct PDF builder using ReportLab is available in rst2pdf version 0.12 or greater. You need to add 'rst2pdf.pdfbuilder' to your extensions to enable it, its name is pdf. Refer to the rst2pdf manual for details.

Could you try that out and report back on whether it works any better?

**Comment by brandon-rhodes** [comment]
I am going to close this for now, because it seems like this is an error with the old way of trying to build PDFs with Sphinx — but if you do run across more information and it looks like PyEphem is using Sphinx wrong, I would be happy to get a pull request showing how the Sphinx config can be updated so that these UTF-8 characters make it through to the LaTeX-generated PDF. In the meantime, thanks for letting me know about this issue in case anyone else runs across it!
